src/renderer/components/+extensions/extensions.tsx

```diff
@@ -269,14 +269,12 @@ export async function attemptInstallByInfo({ name, version, requireConfirmation
   const disposer = ExtensionInstallationStateStore.startPreInstall();
   const registryUrl = new URLParse("https://registry.npmjs.com").set("pathname", name).toString();
   const { promise } = downloadJson({ url: registryUrl });
+  const json = await promise.catch(console.error);
 
-  let json;
+  if (!json || json.error || typeof json.values !== "object" || !json.values) {
```

Comment:
this seems to be blocking valid extensions now too:

<img width="1079" alt="Screen Shot 2021-04-09 at 3 57 45 PM" src="https://user-images.githubusercontent.com/40840436/114234418-b3eab600-994c-11eb-875d-7901b0a4425e.png">

(same for protocol handler)